SPECS/openssl/0001-Replacing-deprecated-functions-with-NULL-or-highest.patch

```diff
@@ -14,61 +14,21 @@ OpenSSL's build time configuration may skip the 'no-<prot>-method'
 switch, while still not supporting the deprecated protocols disabled
 through the 'no-<prot>' switch.
 
-For deprecated TLS protocol versions behind the scenese we're calling
-into 'TLS_(client_|server_)?method()' set of methods, which
+For deprecated DTLS protocol versions behind the scenes we're calling
+into 'DTLS_(client_|server_)?method()' set of methods, which
 automatically negotiate the highest supported protocol.
 
 For SSLv3 methods we're returning a NULL pointer as there are no
 more supported methods for the SSL protocol.
 ---
- ssl/methods.c | 30 +++++++++++++++++++++---------
- 1 file changed, 21 insertions(+), 9 deletions(-)
+ ssl/methods.c | 18 +++++++++++++++---
+ 1 file changed, 15 insertions(+), 3 deletions(-)
 
 diff --git a/ssl/methods.c b/ssl/methods.c
-index c5e8898..90327d7 100644
+index c846143277..a7ae074bfd 100644
 --- a/ssl/methods.c
 +++ b/ssl/methods.c
```

Comment:
All of the Lines below are being deleted. It's confusing for me that some of them are shown in green by the github UI. For example, the green "return TLS_method();" line below is actually being deleted here.